Automattic/simplenote-ios
# NoteListViewController: Implementing UIContextualActions

### Details
- Updates **UITableViewRowAction > UIContextualAction**
- Updates several assets!
- New **Copy Internal Link** Swipeable Action
- Added missing Tracker invocations

@aerych sir!! may I bug you yet again?
Thaaaank youuu!

Ref. #863

### Test: Interlink
1. Open the Notes List
2. Swipe over any row
3. Press the Link button (purple!)
4. Press over any other note

- [x] Verify that you can **paste** a Markdown Link to the selected note
- [x] Verify that the Markdown Link looks as follows: `[Title](simplenote://note/HASH)`

### Test: Regressions
- [x] Verify you can Pin / Unpin a Note by swiping over its row (Notes List!)
- [x] Verify you can Trash a Note by swiping over its row (Notes List!)
- [x] Verify you can Share a Note by swiping over its row (Notes List!)

### Release
`RELEASE-NOTES.txt` was updated in d2c3ae3 with:
 
> Swiping a Note in the List now reveals the Copy Internal Link Action

### Screenshots
<img width="300" src="https://user-images.githubusercontent.com/1195260/92038268-8223c800-ed49-11ea-8551-0bd4cfb8ac7c.png">
